pkgs/applications/networking/browsers/firefox/enterprise-policies.patch

```diff
@@ -0,0 +1,29 @@
+diff --git a/toolkit/components/enterprisepolicies/EnterprisePolicies.js b/toolkit/components/enterprisepolicies/EnterprisePolicies.js
```

Comment:
I'd love to see a bit more description as a comment before the actual diff. In a year from now it should still be crystal clear what this does. Reading in the diff is possible but probably harder than it should be.